script/Ciel/Day 14/14_01B_03_CIEL14B_5B.txt

```diff
@@ -1081,50 +1266,53 @@ I only need to touch it once. I have no choice but to pass it as I fall.
 [sha:b3d6dfe62fcd71745babd34267f5606c73b9d6b9]{
 -- Page 281, Offset 41074.
 -- ―――ナイフを握る右手を胸の位置に構える。
-―――I position the knife gripped on my right hand around my chest.
+―――I hold the knife gripped in my right hand level with my chest.
 }
 [sha:e4385d00543529b6bc99dd4489c8bfc707e8bf68]{
 -- Page 282, Offset 41075.
 -- ―――心臓、というのはあながち間違いではない。
-―――Calling it a heart is not necessarily wrong.
+―――It isn't exactly %{i}wrong%{/i} to call that thing a 'heart'.
 }
 [sha:f40f4ec05a9438f55b70ff63e0e81264833ceca7]{
 -- Page 282, Offset 41076.
 -- 死の点も、死の線も見えない。
-I can't see any lines or points of death.
+I can't see any lines or points of death on it.
 }
 [sha:7bc57278de9697561bbcb9cfff3802d3c0fea62d]{
 -- Page 282, Offset 41077.
 -- わきたつ生命の鼓動。
-The heartbeat of excited life.
+The vigorous heartbeat of life.
 }
 [sha:cdaf338427ee0c1e7610b5c6b1c6b03f2424a15c]{
 -- Page 282, Offset 41078.
 -- 全ての源にして、最後に還りつく星の墓所。
-The graveyard of the planet; the final return to the source of all things.
+The Graveyard of the Planet, from which all originates, and to which all returns.
```

Comment:
Is it planet? Is it star? Who knows. Probably shouldn't be capitalized though. This has never come up before.
>The cemetery of stars, from which all originates...